components/infobox/wikis/stormgate/infobox_unit_custom.lua

```diff
@@ -237,6 +237,8 @@ function CustomUnit:setLpdbData(args)
 			passive = Array.parseCommaSeparatedString(args.passive),
 			armortypes = Array.parseCommaSeparatedString(args.armor_type),
 			upgradesto = Array.parseCommaSeparatedString(args.upgrades_to),
+			bountyluminite = tonumber(args.bounty_luminite),
```

Comment:
no display for these new values needed?

Reply:
I guess we don't want to show the bounty in the unit navbox since the bounty is seperated from the unit and only relevant when part of a camp.